### PR TITLE
G7/feat/tuto

### PR DESCRIPTION
**Pull request by FerreiraLuciano**

### CODE REVIEW

**Review by TheoLandemaine — APPROVED** [review]
👍

File: front-end/src/components/blog/CardArticle.vue
Comment (AyakorK) [inline front-end/src/components/blog/CardArticle.vue:1]: Why did you switch in english while the platform is in french ?

File: front-end/src/components/blog/MarkdownViewer.vue
Comment (AyakorK) [inline front-end/src/components/blog/MarkdownViewer.vue:1]: Remove the comments please

File: front-end/src/components/blog/test_markdown_it.vue
Comment (AyakorK) [inline front-end/src/components/blog/test_markdown_it.vue:1]: ?????????????????

**Review by AyakorK — CHANGES_REQUESTED** [review]

**Review by AyakorK — APPROVED** [review]

**Pull request merged**